src/components/Filters/FilterBlock.vue

```diff
@@ -9,7 +9,7 @@
             @click="onClickIsolatedFilter"><i class="icon cross" /></span>
     </button>
     <button v-else
-            class="filter-block button tiny tag margin-horizontal-smaller"
+            class="filter-block button tiny tag margin-horizontal-smaller margin-top-smaller"
```

Comment:
Adding `margin-top` makes the button misaligned with the `Filter By` text (see screenshot).
It's better to add `margin-bottom`

Reply:
This works